### DISCUSSION TITLE
Frontend Display Dashboard

### DISCUSSION THREAD

**Discussion by AimanFariz** [Dev4w4n/e-masjid.my#146] [Ideas]
Memeriahkan dashboard pengguna dengan elemen-elemen berinformasi

**Comment by AimanFariz** [comment]
Dashboard sedia hanya memaparkan statistik jumlah kutipan Tabung.

Saya ingin mencadangkan penambahan elemen yang memberikan informasi kepada pengguna, iaitu scrollable view untuk :
- imam dan bilal waktu solat seterusnya
- penceramah jemputan (kalau ada)
demo scrollable view:
https://github.com/Dev4w4n/e-masjid.my/assets/99475959/511efe7e-33fb-46ae-8ae2-e806f6ab4f98

**Comment by imie** [comment]
Dekat bahagian tetapan ada pilihan zon waktu, adakah akan di implement waktu solat dan sahur/berbuka pada bahagian hadapan page pada masa akan datang mengikut zon yang telah ditetapkan?

![image](https://github.com/Dev4w4n/e-masjid.my/assets/2321338/18e52b25-1bb6-4638-8c08-5bfc7c2948ef)

![image](https://github.com/Dev4w4n/e-masjid.my/assets/2321338/7526740f-a040-4528-973c-4f2394c6bcd2)

**Reply by rohaizanr** [reply]
ya @imie , cadangnya macam tu la. ada satu API belum siap
